requirements.txt

```diff
@@ -0,0 +1,8 @@
+nltk
+scikit-learn
+pandas
+h5py
+cython
+tqdm
+numpy
+torch=1.0.0
```

Comment:
Sorry I forgot to mention this in my last review.  This should be `==` instead of `=`